src/snowcli/cli/nativeapp/commands.py

```diff
@@ -71,3 +75,18 @@ def app_bundle(
     manager = NativeAppManager(project_path)
     manager.build_bundle()
     return MessageResult(f"Bundle generated at {manager.deploy_root}")
+
+
+@app.command("diff", hidden=True)
+@with_output
+@global_options_with_connection
+def nativeapp_stage_diff(
```

Comment:
other functions in this file have names like `app_command`

is there any reason why this function does not follow this convention?